src/js/pureFullPage.js

```diff
@@ -140,6 +140,15 @@ class PureFullPage {
       this.goUp();
     }
   }
+
+  handleMouseWheel = null;
+  handleNewPosition = null;
```

Comment:
How about add this two functions into `constructor`?

把这两个函数放到 `constructor` 中会不会更好?

 ```suggestion
 constructor (options) {
   // ...
   this.handleMouseWheel = null;
   this.handleNewPosition = null;
   // ...
 }

```